src/services/users/index.js

```diff
@@ -182,11 +184,41 @@ const getNewTokens = async (email, currentRefreshToken) => {
 	return { accessToken, refreshToken, userId: user.id }
 }
 
+const getInvitedUserDetail = async (token) => {
+	try {
+		const decoded = jwt.verify(token, config('TOKEN_SECRET'))
+		const {email} = decoded
+		const result = await  db.query(
+			`SELECT * FROM user_invites
+		WHERE email = $1 
+		`,
+			[email]
+		)
+		const user = result.rows[0]
+		if (!user || user.status === "active") {
+			throw customError(InvalidInviteError)
+		}
+		const accessToken = jwt.sign({email}, 
```

Comment:
kindly use the `generateAccessToken` function

Reply:
i have gotten it done